### ISSUE TITLE
Show that the CRDTs work across machines. 

### ISSUE THREAD

**Issue by russwyte** [achanda/crdt-rs#1]
Forgive me if I am missing something basic (a very really possibility I am a Rust noob) but I don't see where the key goal (or at least one of the most practical) is achieved...

"CRDTs are used to replicate data across multiple computers in a network"

I can't see where there is any such replication across a network in the example.

This is a very cool idea though - and I would love to see where this goes!

Cheers!

**Comment by achanda** [comment]
@russwyte you're correct that these are not exactly serializable across a network at this moment. That's why the examples say nothing about that. However, that should be pretty easy to implement, let me get that done soon.
